src/best-practices/caddy.rst

```diff
@@ -0,0 +1,167 @@
+.. Licensed under the Apache License, Version 2.0 (the "License"); you may not
+.. use this file except in compliance with the License. You may obtain a copy of
+.. the License at
+..
+..   http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+.. WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+.. License for the specific language governing permissions and limitations under
+.. the License.
+
+.. _best-practices/Caddy:
+
+========================
+Caddy as a Reverse Proxy
+========================
+
+CouchDB recommends the use of `HAProxy`_ as a load balancer and reverse proxy.
+The team's experience with using it in production has shown it to be superior
+for configuration and monitoring capabilities, as well as overall performance.
+
+CouchDB's sample haproxy configuration is present in the `code repository`_ and
+release tarball as ``rel/haproxy.cfg``.
+
+However, ``Caddy`` is a suitable alternative. Below are instructions on
+configuring Caddy appropriately.
```

Comment:
Since you're copying over the content from `src/best-practices/nginx.rst`, why not rename that file to `src/best-practices/reverse-proxy.rst` and just add Caddy below the Nginx section? I don't like the idea of having to maintain both of these files in the doc tree in case the haproxy.cfg file changes, or our feelings about HAProxy itself changes.

Reply:
Nice idea! Merged nginx and Caddy into section 'Reverse Proxies' - https://github.com/apache/couchdb-documentation/pull/318/commits/a7bd43ad8d7a4fe2adb2f0c7586203bdf7562f24